src/node_process_object.cc

```diff
@@ -105,18 +105,37 @@ MaybeLocal<Object> CreateProcessObject(Realm* realm) {
   READONLY_PROPERTY(
       process, "version", FIXED_ONE_BYTE_STRING(isolate, NODE_VERSION));
 
-  // process.versions
   Local<Object> versions = Object::New(isolate);
-  READONLY_PROPERTY(process, "versions", versions);
+  // node.js version is always on the top
+  READONLY_STRING_PROPERTY(
+      versions, "node", per_process::metadata.versions.node);
+  std::vector<std::pair<const char*, std::string>> versions_vec;
```

Comment:
```suggestion
  std::vector<std::pair<std::string_view, std::string_view>> versions_vec;
```

Changing the first part allows you to use the standard `<` for instead of `strcmp` below and is generally a bit more C++-y, changing the second parts avoids a copy that we don’t need here :slightly_smiling_face: 

EDIT: Actually … if you use `std::map<std::string_view, std::string_view>`, I _think_ this would actually all work out on its own because `std::map` orders its entries by default?